examples/app-with-on-behalf-of-auth/docker-compose.yaml

```diff
@@ -6,9 +6,12 @@ services:
     build:
       context: ./.config
       args:
-        grafana_version: ${GRAFANA_VERSION:-9.5.3}
+        # Points to an unreleased v10.1 version of Grafana
+        grafana_version: ${GRAFANA_VERSION:-main}
```

Comment:
is `grafana_image` applicable when on unreleased versions?

Reply:
the `grafana_image` is the same (always `grafana/grafana`)